Oswaldo Vazquez Aparicio/30 Agosto/hackerrank/JavaHashSet.java

```diff
@@ -10,7 +11,7 @@ public static void main(String[] args) {
         Scanner scanner = new Scanner(System.in);
         int pairsNumber = scanner.nextInt();
 
-        HashSet<String> pairs = new HashSet<>();
+        Set<String> pairs = new HashSet<>();
```

Comment:
When you variable is a Collection, it is good practice put name in singular and them the the type, example: tableListList, userSet, etc...